CMakeLists.txt

```diff
@@ -70,7 +70,7 @@ if (NOT CMAKE_BUILD_TYPE)
 endif()
 
 if (NOT DEFINED AWS_CRT_CPP_VERSION)
-    set(AWS_CRT_CPP_VERSION "v0.17.30")
+    set(AWS_CRT_CPP_VERSION "0.17.30")
```

Comment:
Don't strip the "v" off here, this string is already being used for metrics
https://github.com/awslabs/aws-crt-cpp/blob/421a69a81fb73b4c9917ffb3d095c2c65e9e973d/source/iot/MqttClient.cpp#L350

Make a new variable that strips off the "v" and use that for package version

Reply:
Fixed, published new version.